pom.xml

```diff
@@ -528,94 +542,44 @@
       </activation>
       <build>
         <pluginManagement>
-          <!--TODO: set up error-prone when it supports JDK 11-->
-          <!--https://github.com/google/error-prone/issues/1091-->
-          <!--https://github.com/google/error-prone/issues/1127-->
           <plugins>
             <plugin>
-              <groupId>org.apache.maven.plugins</groupId>
               <artifactId>maven-compiler-plugin</artifactId>
               <version>3.8.0</version>
               <configuration>
-                <release>8</release>
+                <source>11</source>
```

Comment:
Move them to `properties` to be more visible? `maven.compiler.source` and such.

Reply:
And perhaps there will be no need to customize this plugin.

Reply:
We still need to customize it in order to configure errorprone and linting etc.

Reply:
Turns out we needed this because the foss-root pom explicitly sets (and thus overrides) this:
```
            <source>1.8</source>
            <target>1.8</target>
```